cards/src/main/java/com/decathlon/vitamin/compose/cards/VitaminCardColors.kt

```diff
@@ -0,0 +1,25 @@
+package com.decathlon.vitamin.compose.cards
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.graphics.Color
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+@Immutable
+data class CardColors(
+    val backgroundColor: Color
```

Comment:
We should add a content color no?

Reply:
We don't really need contentColor as we don't provide any content, only the container. Or do you see contentColor for something else ?